test/unit/bps_tree_generic.cc

```diff
@@ -4,6 +4,10 @@
 #include <inttypes.h>
 #include <time.h>
 
+#ifdef BPS_BLOCK_CHILD_CARDS_ARRAY
+#define UNIT_TAP_COMPATIBLE 1
+#endif
```

Comment:
Looks like a good change, even if without the rest of the patch. Please consider patching the tests in a separate commit.

Reply:
Moved this into a separated commit.